docs/azure-openshift.md

```diff
@@ -37,10 +37,15 @@ openshift-console          downloads           downloads-openshift-console.apps.
 
 6. Get the list of dns zones w.r.t your resource group to find the one which corresponds to the previously found route’s domain:
 ```bash
```

Comment:
Add a header similar to this, to distinguish this from the query for private dns zones.
I'm not sure what they should do if there are both public and private dns zones.  Is it possible?  
```suggestion
For public dns zones:
```bash
```